ts/client/src/client.ts

```diff
@@ -1280,6 +1329,28 @@ export class MangoClient {
     return await this.sendAndConfirmTransactionForGroup(group, [ix]);
   }
 
+  public async serum3EditMarket(
+    group: Group,
+    serum3MarketIndex: MarketIndex,
+    reduceOnly: boolean,
+    forceClose: boolean,
+  ): Promise<TransactionSignature> {
+    const serum3Market =
+      group.serum3MarketsMapByMarketIndex.get(serum3MarketIndex);
+    const ix = await this.program.methods
+      .serum3EditMarket(
+        reduceOnly != null ? reduceOnly : null,
```

Comment:
is it really allowed to pass `null` to a function expecting `boolean`?

Reply:
Also, this check is weird. The expression is different from just `reduceOnly` because so many things compare positive to `null` in js, but the intent of the code is still unclear to me. I'd rather if stuff like  reduceOnly=0 or reduceOnly=[] be an error.

Reply:
the ix is expecting a Option<bool>, not just a boolean, is that what you meant?

I have now changed it to, wdyt?
![image](https://user-images.githubusercontent.com/89031858/233585000-81766cb2-ff2e-491d-8be5-abeba2bb38f0.png)

Reply:
I don't get it. What is `reduceOnly ?? reduceOnly` supposed to do? Why not just `reduceOnly`?

Reply:
ok it should be `reduceOnly ?? null`, which accounts for undefined been passed